common/components/time/template.test.js

```diff
@@ -60,4 +69,80 @@ describe('Time component', function () {
       expect($component.text().trim()).to.equal('Today')
     })
   })
+
+  context('with relative time', function () {
+    let $component
+
+    beforeEach(function () {
+      const $ = renderComponentHtmlToCheerio('time', {
+        datetime: '2019-01-10',
+        relative: true,
+      })
+
+      $component = $('time')
+    })
+
+    it('should contain a datetime attribute', function () {
+      const $datetimeAttr = $component.attr('datetime')
+      expect($datetimeAttr).to.equal('2019-01-10')
+    })
+
+    it('should contain a display relative attribute', function () {
+      expect($component.attr('data-display-relative')).to.equal('true')
+    })
+  })
+
+  context('with imminent offset', function () {
+    let $component
+
+    beforeEach(function () {
+      const $ = renderComponentHtmlToCheerio('time', {
+        datetime: '2019-01-10',
+        imminentOffset: 120,
+      })
+
+      $component = $('time')
+    })
+
+    it('should contain a datetime attribute', function () {
+      const $datetimeAttr = $component.attr('datetime')
+      expect($datetimeAttr).to.equal('2019-01-10')
+    })
+
+    it('should contain a display relative attribute', function () {
+      expect($component.attr('data-imminent-offset')).to.equal('120')
+    })
+  })
+
+  context('shown as a tag', function () {
```

Comment:
The other two contexts are of the form `with display as tag` - it might be clearer to have all these contexts match.